monopoly/Player.js

```diff
@@ -21,7 +28,49 @@ class Player {
     this.getOutCard = getOutCard;
     this.jail = jail;
     this.number = number;
-    this.playerRoll = false; // TODO - Does this need to stored within the class?
-    this.doubleCounter = 0; // TODO - Does this need to stored within the class?
+    this.playerRoll = false;
+    this.doubleCounter = 0;
+    this.moveType = moveType;
+    this.moveTo = moveTo;
+    this.collect = collect;
+    this.buyable = buyable;
+    this.payMultiplier = payMultiplier;
+    this.houseRepairCost = houseRepairCost;
+    this.hotelRepairCost = hotelRepairCost;
+  }
+
+  ProcessCard(card) {
```

Comment:
Can you change this to be a switch case instead of if elses. I think that will make this a lot easier to modify down the road.